statistician/requirements.txt

```diff
@@ -1,9 +1,8 @@
 --extra-index-url https://packages.dea.ga.gov.au/
 
-datacube[performance,s3]==1.8.5
-aiobotocore[awscli,boto3]==1.3.3
+datacube[performance,s3]
+aiobotocore[awscli,boto3]
 odc-stats
-hdstats==0.1.8.post1
```

Comment:
Why unpin here?

Reply:
by default hdstats install 0.2.1 even you pin it here so I have to add that additional step

Reply:
Strange. Ok!